provider/resource_keycloak_realm.go

```diff
@@ -441,6 +441,148 @@ func resourceKeycloakRealm() *schema.Resource {
 				Type:     schema.TypeMap,
 				Optional: true,
 			},
+
+			// WebAuthn
+			"web_authn_policy": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"acceptable_aaguids": {
+							Type: schema.TypeList,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+							Optional: true,
+						},
+						"attestation_conveyance_preference": {
+							Type:        schema.TypeString,
+							Description: "Either none, indirect or direct",
+							Optional:    true,
+							Default:     "not specified",
```

Comment:
I really hate how the Keycloak API uses this as the nil value.  I'm not sure how I feel about setting this as the default within the provider package.  However, I don't feel strongly enough about this to force you to change it.

Reply:
The javascript code check if it is `not specfied`, so... :/

https://github.com/keycloak/keycloak/blob/e239699b8199dc1997227c9caef15115db9eb8e5/themes/src/main/resources/theme/base/login/webauthn-register.ftl#L52

Reply:
We could use the WebAuthn defaults though. Ex for `attestation_conveyance_preference` [the default is `none`](https://w3c.github.io/webauthn/#dom-publickeycredentialcreationoptions-attestation), any opinions?